### ISSUE TITLE
Please add the ability to disable fish and use bash from as a choice in the installer

### ISSUE THREAD

**Issue by ryzendew** [CachyOS/cachyos-calamares#30]
Look i hate fish sooooo much it's weird and clunky and sets off my mental state can you please let us use bash as a choice in the install cause unpicking fish does nothing! please i beg of you ... i don't want to suffer any longer.

Sent with love from the youtuber that whores you out Daily mattscreative

**Comment by EnderSpy29** [comment]
Yes! i don't want fish on my system, unchecking it should remove it and make Bash the default shell.

Even better. Allow us to pick a preferred shell such as ZSH, Fish, Bash, ZSH+OhMyZSH+Archlinux Plugin and so on

**Comment by praetorxyn** [comment]
I use fish, but I think it would be better to have one of the installer steps being selecting the preferred shell, and just having one of them selected by default (whether it's bash for mass appeal and familiarity or fish because it's likely the best for new users).

**Comment by EnderSpy29** [comment]
> I use fish, but I think it would be better to have one of the installer steps being selecting the preferred shell, and just having one of them selected by default (whether it's bash for mass appeal and familiarity or fish because it's likely the best for new users).

should be Bash. Bash just works. on every system everywhere

**Comment by ventureoo** [comment]
Implemented in https://github.com/CachyOS/cachyos-calamares/pull/45, so I think this can be closed.
